socorro/unittest/external/es/test_index_cleaner.py

```diff
@@ -97,17 +93,43 @@ def test_delete_old_indices(self):
         assert not self.index_client.exists('test_socorro200201')
 
     @minimum_es_version('1.0')
-    def test_other_indices_are_not_deleted(self):
+    def test_delete_old_indices_other_indices_are_not_deleted(self):
         """Verify that non-week-based indices are not removed.
         """
         # Create a temporary index.
         self.index_creator.create_index('socorro_test_temp', {})
-        self.indices.append('socorro_test_temp')
-
-        assert self.index_client.exists('socorro_test_temp')
 
         api = IndexCleaner(self.config)
         api.delete_old_indices()
 
         # Verify the email index is still there.
         assert self.index_client.exists('socorro_test_temp')
+
+    @minimum_es_version('1.0')
```

Comment:
We never run with ES < 1.0. We can nix these decorators and any others like this.

Reply:
We can nix the decorators in another PR--doesn't have to be this one. One of us is probably going to be doing some ES work in the near future, so we can change things then.

Reply:
I can at least remove them from this test file.